src/OrbitGl/TimelineTicks.cpp

```diff
@@ -83,7 +85,14 @@ int TimelineTicks::GetTimestampNumDigitsPrecision(uint64_t timestamp_ns) const {
   return kMaxDigitsPrecision;
 }
 
-uint64_t TimelineTicks::GetScale(uint64_t visible_ns) const {
+uint64_t TimelineTicks::GetMinorTicksScale(uint64_t visible_ns) const {
+  uint64_t major_scale = GetMajorTicksScale(visible_ns);
+  // For consistency, minor ticks scale is the next finer scale than the one used for major ticks.
```

Comment:
nit. "the next finer scale *after* the one used for major ticks." would be more readable imo.